src/main/java/mate/academy/spring/controller/AuthenticationController.java

```diff
@@ -0,0 +1,28 @@
+package mate.academy.spring.controller;
+
+import mate.academy.spring.model.User;
+import mate.academy.spring.model.dto.request.UserRequestDto;
+import mate.academy.spring.model.dto.response.UserResponseDto;
+import mate.academy.spring.security.AuthenticationService;
+import mate.academy.spring.service.dto.mapping.DtoResponseMapper;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class AuthenticationController {
+    private final DtoResponseMapper<UserResponseDto, User> dtoResponseMapper;
+    private final AuthenticationService authenticationService;
+
+    public AuthenticationController(DtoResponseMapper<UserResponseDto,
+            User> dtoResponseMapper, AuthenticationService authenticationService) {
+        this.dtoResponseMapper = dtoResponseMapper;
+        this.authenticationService = authenticationService;
+    }
+
+    @PostMapping("/register")
+    public UserResponseDto register(@RequestBody UserRequestDto userRequestDto) {
```

Comment:
use @Valid annotation. Spring should move this object to validator, before doing something